en/getting_started/basic_flight_modes.md

```diff
@@ -2,185 +2,178 @@
 
-**ALTITUDE: Altitude Control**
+![MC Altitude Mode](../../images/flight_modes/altitude_MC.png)
 
-Altitude Mode is the safest non GPS guided mode appropriate for beginners learning how to fly. In this mode the altitude is controlled by the PX4 autopilot. At 50% throttle the aircraft will hold its current altitude. If you want to descend just lower the throttle bellow center or if you need to climb push the throttle over the center. The further the throttle stick is from the center position the larger the climb or descent rate you will have. In Altitude Mode you have full angle control over Pitch, Roll and Yaw. Once you center all the sticks you will have a hovering aircraft at the current altitude. If the wind blows the aircraft will drift in the direction of the wind.
+#### Position Mode
 
-![MC Altitude Mode](../../images/flight_modes/altitude_MC.png)
- 
- 
-![GPS](../../images/flight_modes/GPS_s.png) **POSITION: Position Control**
+![GPS](../../images/flight_modes/GPS_s.png) Position mode is the GPS guided mode. 
 
-This is the GPS guided mode. You can fly in Position Mode if you need precision in flight, keeping course or hovering at a certain spot in the air, but still maintaining the control of the aircraft with your RC. The altitude is controlled by the PX4 autopilot. At 50% throttle the aircraft will hold its current altitude. If you want to descend just lower the throttle bellow center or if you need to climb push the throttle over the center. The further the throttle stick is from the center position the larger the climb or descent rate you will have. Position Mode is almost like Altitude Mode with full angle control over Pitch, Roll and Yaw. But Once you center all the sticks you will have a hovering aircraft at the current altitude and current GPS position. If the wind blows the aircraft will keep its direction of flight and course and will fight the wind.
+You can fly in Position Mode if you need precision in flight, keeping course or hovering at a certain spot in the air, but still maintaining the control of the aircraft with your RC. The altitude is controlled by the PX4 autopilot. At 50% throttle the aircraft will hold its current altitude. If you want to descend just lower the throttle bellow center or if you need to climb push the throttle over the center. The further the throttle stick is from the center position the larger the climb or descent rate you will have. Position Mode is almost like Altitude Mode with full angle control over Pitch, Roll and Yaw. But once you center all the sticks you will have a hovering aircraft at the current altitude and current GPS position. If the wind blows the aircraft will keep its direction of flight and course and will fight the wind.
 
 ![MC Position Mode](../../images/flight_modes/position_MC.png)
  
  
-## Autonomous Modes
+### Autonomous Modes
 
 Autonomous or Auto modes do not require any input from the pilot.
  
-**PAUSE: Auto Pause**
+#### Pause (Auto Pause)
```

Comment:
- @tubeme We will have to rename this to Hold - that is the name of the mode in QGC.

Reply:
The code is still messy at some places. The name of this mode is Pause, And the voice is calling "Auto Pause activated" so I don't think Hold is correct. I will check it how it behaves with the qgc 3.3 and we will decide. (I initiated the concept to change the names so there were very long discussions long time ago about this names...)

Reply:
It was Pause the moment I wrote this doc long time ago. And it was Pause up until 3.2.5. Now I see they changed it indeed and it slipped my attention.